app/graphql/types/query_type.rb

```diff
@@ -1,13 +1,23 @@
 module Types
   class QueryType < Types::BaseObject
-    # Add root-level fields here.
-    # They will be entry points for queries on your schema.
+    field :all_markets, [Types::MarketType], null: false
+    def all_markets
+      Market.all
+    end
+
+    # Save for future use to search markets by products, etc.
+    # 
+    # field :all_markets, [Types::MarketType], null: false
+    # def all_markets
+    #   Market.all
+    # end
 
-    # TODO: remove me
-    field :test_field, String, null: false,
-      description: "An example field added by the generator"
-    def test_field
-      "Hello World!"
+    field :market, Types::MarketType, null: false do
+      argument :id, Integer, required: true
+    end
+    def market(id)
+      Market.find(id[:id])
     end
```

Comment:
We might want to change this to be more universal and search by `fmid` instead of our database id (or just add `fmid` as an additional optional argument, ie on line 16.5: `argument :fmid, Integer, required: false`)

Reply:
Agreed we should accept multiple fields, name would be a good one too